independent-projects/parent/pom.xml

```diff
@@ -720,6 +708,36 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <configuration>
+                            <archive>
+                                <manifestEntries combine.children="append">
```

Comment:
For reviewer: this part is in the `release` profile so will be enabled only when releasing Quarkus.